e2e/wdio/platform/pages/select.po.ts

```diff
@@ -0,0 +1,44 @@
+import { BaseComponentPo } from './base-component.po';
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class SelectPo extends BaseComponentPo {
+    url = '/select';
+    root = '#page-content';
```

Comment:
just for future, you can remove 'root' from here cause this variable is already declared in CoreBaseComponentPo

Reply:
@maxym-valor if not requesting a change, let's resolve this comment to not block the PR from merging.